bin/check-exercise.sh

```diff
@@ -144,4 +144,12 @@ check_format tests "${exercise_path}/${exercise_name}_test.odin"
 check_format example "${exercise_path}/.meta/example.odin"
 check_for_missing_test_descriptions "${exercise_path}/${exercise_name}_test.odin"
 
+    if [ -f ODIN_VERSION ]; then
+        local_version=$(odin version | sed -E 's/:.*$//' | sed -E 's/^.*odin version //')
```

Comment:
No need to call sed twice
```suggestion
        local_version=$(odin version | sed -E 's/:.*$//; s/^.*odin version //')
```